_includes/pbc/all/install-features/202404.0/install-dynamic-multistore.md

```diff
@@ -0,0 +1,1395 @@
+        ];
+    }
+}
+```
+{% info_block warningBox "Verification" %}
+
+Please make sure that the following configuration is working via run `vendor/bin/console queue:setup` command.
+
+Make sure the `sync.storage.store` queue exists in RabbitMQ.
+
+{% endinfo_block %}
+
+
+
+### 2) Set up the database schema and transfer objects
+
+1. Adjust the schema definition so entity change triggers events:
+
+**src/Pyz/Zed/Country/Persistence/Propel/Schema/spy_country.schema.xml**
+
+```xml
+
+<?xml version="1.0"?>
+<database xmlns="spryker:schema-01" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed"
+          xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd"
+          namespace="Orm\Zed\Country\Persistence" package="src.Orm.Zed.Country.Persistence">
+
+    <table name="spy_country_store" idMethod="native">
+        <behavior name="event">
+            <parameter name="spy_country_store_all" column="*"/>
+        </behavior>
+    </table>
+
+</database>
+
+```
+
+**src/Pyz/Zed/Locale/Persistence/Propel/Schema/spy_locale.schema.xml**
+
+```xml
+<?xml version="1.0"?>
+<database xmlns="spryker:schema-01" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed"
+          xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd"
+          namespace="Orm\Zed\Locale\Persistence" package="src.Orm.Zed.Locale.Persistence">
+
+    <table name="spy_locale_store" idMethod="native">
+        <behavior name="event">
+            <parameter name="spy_locale_store_all" column="*"/>
+        </behavior>
+    </table>
+
+</database>
+
+```
+
+**src/Pyz/Zed/Store/Persistence/Propel/Schema/spy_store.schema.xml**
+
+```xml
+<?xml version="1.0"?>
+<database xmlns="spryker:schema-01" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed"
+          xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd"
+          namespace="Orm\Zed\Store\Persistence" package="src.Orm.Zed.Store.Persistence">
+
+    <table name="spy_store">
+        <behavior name="event">
+            <parameter name="spy_store_all" column="*"/>
+        </behavior>
+    </table>
+
+</database>
+```
+
+2. Apply database changes and generate entity and transfer changes:
+
+```bash
+console propel:install
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the following changes have been applied in the database:
+
+| DATABASE ENTITY                       | TYPE   | EVENT   |
+|---------------------------------------|--------|---------|
+| spy_store.fk_locale                   | column | added   |
+| spy_country_store                     | table  | added   |
+| spy_locale_store                      | table  | added   |
+
+{% endinfo_block %}
+
+
+{% info_block warningBox "Verification" %}
+
+
+1. Stop the scheduler:
+
+```bash
+vendor/bin/console scheduler:suspend
+```
+
+2. In the Back Office, set up a store with a country and locale.
```

Comment:
First, you can check it in the back office after saving the form.
Then after the last section in the documentation when all frontend and infrastructure changes will be applied, it will be visible, for example, on Yves. 
We may not have to add any additional verification steps here.